lib/mayaUsd/ufe/Global.cpp

```diff
@@ -134,6 +137,8 @@ std::string defaultMaterialsScopeName()
     return UsdMayaJobExportArgs::GetDefaultMaterialsScopeName();
 }
 
+const char* getTransform3dMatrixOpName() { return std::getenv("MAYA_USD_MATRIX_XFORM_OP_NAME"); }
```

Comment:
I didn't change this code - it just moved here from the transform code as I needed to make it only for MayaUsd.